lib/spotlight/dor/indexer.rb

```diff
@@ -83,6 +91,31 @@ def add_point_bbox(sdb, solr_doc)
       def add_series(sdb, solr_doc)
         solr_doc['series_ssi'] = sdb.smods_rec.series
       end
+
+      # @return [Array{String}] The IDs from geonames //subject/geographic URIs, if any
+      def extract_geonames_ids(sdb)
+        sdb.smods_rec.subject.map do |z|
+          uri = z.geographic.attr('valueURI')
+          m = %r{^https?:\/\/sws\.geonames\.org\/(\d+)}i.match(uri.nil? ? '' : uri.value)
```

Comment:
Do we need the escaped slashes? I thought `%r{}` was used to get us out of that mess?